CIP-0060/README.md

```diff
@@ -0,0 +1,126 @@
+---
+CIP: 60
+Title: Music Token Metadata
+Authors: Andrew Westberg <awestberg@projectnewm.io>, Ryan Jones <rjones@projectnewm.io>, Justin Morgan <jusemorgan@gmail.com>, Ian Singer <tcl@fre5hmusic.com>, Anthony Eizmendiz <aeizmendiz@icloud.com>, Session Cruz <session@demu.pro>, Jimmy Londo <SickCityCleveland@gmail.com>, Gudbrand Tokerud <Gudbrand.tokerud@gmail.com>, Kevin St.Clair <kos1777@gmail.com>
+Comments-URI: no comments yet
+Status: Draft
+Type: Informational
+Created: 2022-07-26
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This proposal defines an extension to CIP-25 for token metadata specific to music tokens.
+
+# Motivation
+
+Music tokens on Cardano can be either NFTs or FTs and contain links to audio files. In order for players, indexers, and wallets to be able to properly search and categorize a user's music collection, we need to define a common schema for creating music on Cardano. If all parties creating these music tokens follow similar patterns, apps can consume this information and make proper use of it. The existing CIP-25 is a good base to build upon, but for a good music experience, we need to standardize additional fields that will be required specifically for music tokens.
+
+# Specification
+
+This CIP divides the additional metadata parameters into two categories of `Required` and `Optional`. When minting a music token on Cardano, you are expected to include ALL of the required fields. If you choose to include one or more of the optional fields, they must be named exactly as defined in this CIP. This will properly allow indexing apps and music players to utilize as much of your token metadata as possible without issues.
+
+## Required Fields ###
+| Field | Type | Example(s) | Notes |
```

Comment:
You may want to consider giving a CDDL definition for the metadata format since that is what is actually used on-chain